### DISCUSSION TITLE
Git push suddenly not triggering deployment

### DISCUSSION THREAD

**Discussion by nalnir** [vercel/community#7377] [Help] [answered]
### Summary

This issue makes me confused as I made no changes to settings in vercel or my github repo. 
I was working today on some new features and everytime a new push to main branch the deployment was working just fine until suddenly when I made another feature it did not trigger the deployment at all. 
I have no idea why. I checked for vercel comment, but there are not comments to my 2 commits I pushed to main branch. It just stopped working out of nowhere.

Once again, I made no changes to anything in my vercel or my github repo so I don't know where to start with debugging really.

### Steps to Reproduce

1. Make a change in code
2. Commit and push
3. Vercel does not build and deploy at all

**Comment by szhu** [comment]
Hi, I found this discussion because I also ran into this issue.

It seems like Vercel is aware of the issue now. They put up a notice:

<img width="1278" alt="image" src="https://github.com/vercel/community/assets/1570168/bc332c7c-62f8-47fb-b794-e646c09180ce">

I tried out the Create Deployment button and it seems to work as a manual workaround for now -- I was able to create a new deployment successfully and it correctly updated the status on GitHub too.

**Reply by leerob** [reply, marked as answer]
We are monitoring the GitHub incident here, but in the meantime, you can create deployments there or through the CLI!

https://www.githubstatus.com/incidents/5yx1d67vq9hg

[a comment by an automated account is not shown]
